### ISSUE TITLE
checkup - Windows Defender ExclusionPath not accesible as normal user in W10 Build 19041

### ISSUE THREAD

**Issue by shacal** [ScoopInstaller/Scoop#4098]
Currently user gets this on every checkup, even if paths are set correctly.

```
❯ scoop checkup --full
WARN  Windows Defender may slow down or disrupt installs with realtime scanning.
  Consider running:
    sudo Add-MpPreference -ExclusionPath 'C:\Users\username\scoop'
  (Requires 'sudo' command. Run 'scoop install sudo' if you don't have it.)
WARN  Windows Defender may slow down or disrupt installs with realtime scanning.
  Consider running:
    sudo Add-MpPreference -ExclusionPath 'C:\ProgramData\scoop'
  (Requires 'sudo' command. Run 'scoop install sudo' if you don't have it.)
WARN  Found 2 potential problems.

~
❯ (Get-MpPreference).exclusionPath
N/A: Must be admin to view exclusions

~
❯ [System.Environment]::OSVersion.Version

Major  Minor  Build  Revision
-----  -----  -----  --------
10     0      19041  0
```

Add a way to manually mark these checks as done / to skip in config or variable.

eg.
`scoop checkup -set ignoredefender`

or/and write down an hint to run checkup like this if "{N/A: Must be admin to view exclusions}" is present.
`sudo scoop checkup`

**Comment by adamvlasak** [comment]
Microsoft Windows [Version 10.0.19041.508]

Yesterday I was reinstalling my system. I am pretty sure that I went through scoop checkup and set all exclusion paths using `sudo Add-MpPreference -ExclusionPath 'C:\Users\adamv\scoop'`.

I can confirm I have correct exclusion paths set from yesterday:

![image](https://user-images.githubusercontent.com/203117/95491234-a18dc080-0999-11eb-84fb-358a815a1ae3.png)

Today I noticed defender is not happy:

```
C:\Users\adamv>sudo Add-MpPreference -ExclusionPath 'C:\ProgramData\scoop'
At line:1 char:1
+ & 'C:\Users\adamv\scoop\apps\sudo\current\sudo.ps1'  Add-MpPreference ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
This script contains malicious content and has been blocked by your antivirus software.
    + CategoryInfo          : ParserError: (:) [], ParentContainsErrorRecordException
    + FullyQualifiedErrorId : ScriptContainedMaliciousContent
```

There is actual defender action happening. It can be seen on the picture.

![image](https://user-images.githubusercontent.com/203117/95491168-8622b580-0999-11eb-956f-4fa0f07d6dfc.png)

Since yesterday I did few restarts.

**Comment by x-magic** [comment]
This happened to my 20H2 installation as well: 
```
PS C:\Users\Bill> scoop checkup
WARN  Windows Defender may slow down or disrupt installs with realtime scanning.
  Consider running:
    sudo Add-MpPreference -ExclusionPath 'C:\Users\Bill\scoop'
  (Requires 'sudo' command. Run 'scoop install sudo' if you don't have it.)
WARN  Windows Defender may slow down or disrupt installs with realtime scanning.
  Consider running:
    sudo Add-MpPreference -ExclusionPath 'C:\ProgramData\scoop'
  (Requires 'sudo' command. Run 'scoop install sudo' if you don't have it.)
WARN  Found 2 potential problems.
PS C:\Users\Bill> (Get-MpPreference).exclusionPath
N/A: Must be admin to view exclusions
PS C:\Users\Bill> [System.Environment]::OSVersion.Version

Major  Minor  Build  Revision
-----  -----  -----  --------
10     0      19042  0
```

Probably has something to do with the fact that `(Get-MpPreference).exclusionPath` requires admin rights now. 

Maybe consider add an ignore switch (like OP suggested), or make this checkup sudo-able?

**Comment by rashil2000** [comment]
Closed in #4699
